layouts/partials/sections/contact.html

```diff
@@ -9,7 +9,7 @@ <h1><span class="icon icon-mail"></span>{{ i18n "letsChat" }}</h1>
     </div>
   </div>
 
-  <form id="form-contact" action="https://formspree.io" method="POST">
+  <form id="form-contact" action="{{site.Params.Feat.ajaxBasin}}" method="POST">
```

Comment:
This is not hardcoded. Instead, it is not used at all and meant to trick spammers (see my comment below). The AJAX Request is handled via JS. Please take a look here:

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/assets/js/src/contact.js#L17-L26